server/command.go

```diff
@@ -150,6 +150,7 @@ func addSubCommands(jira *model.AutocompleteData, optInstance bool) {
 	jira.AddCommand(createWebhookCommand(optInstance))
 
 	// Help and info
+	jira.AddCommand(model.NewAutocompleteData("setup", "", "Start Jira plugin setup flow"))
 	jira.AddCommand(model.NewAutocompleteData("info", "", "Display information about the current user and the Jira plug-in"))
 	jira.AddCommand(model.NewAutocompleteData("help", "", "Display help for `/jira` command"))
```

Comment:
Since this new command is an admin command, I think this should instead be placed into the section above called `Admin Commands`